CHANGELOG.md

```diff
@@ -8,11 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased][]
 
 ### Deprecated
-
-* Support for Font Awesome icons (the `faIcon` parameter in `portlet-definition`
```

Comment:
What's up with no longer wrapping this bullet at 80 characters?

Reply:
Doing whatever I can to quell the incessant nagging of our markdown linter :/